cli.go

```diff
@@ -239,14 +245,17 @@ Configuration:
   role=<string>
       Vault role of type "OIDC" to use for authentication.
 
+  listenaddress=<string>
+    Optional address to bind the OIDC callback listener to (default: localhost).
+
   port=<string>
-	  Optional localhost port to use for OIDC callback (default: 8250).
+    Optional localhost port to use for OIDC callback (default: 8250).
 
   callbackmethod=<string>
-	  Optional method to to use in OIDC redirect_uri (default: http).
+    Optional method to to use in OIDC redirect_uri (default: http).
 
   callbackhost=<string>
-	  Optional callback host adddress to use in OIDC redirect_uri (default: localhost).
+    Optional callback host address to use in OIDC redirect_uri (default: localhost).
```

Comment:
the help string had a mix of tabs and spaces towards the end, which I updated to be more consistently spaces